RHOSO-18.0/manila/powerscale/README.md

```diff
@@ -0,0 +1,207 @@
+            emc_share_backend = isilon
+            share_backend_name = powerscale
+            enabled_share_protocols = NFS,CIFS
+            share_driver = manila.share.drivers.dell_emc.driver.EMCShareDriver
+            emc_nas_root_dir = /ifs/manila
+            emc_ssl_cert_verify = False
+            emc_nas_server_port = 8080
+          customServiceConfigSecrets:
+          - manila-powerscale-secrets
+          networkAttachments:
+          - storage
+...
+```
+* Save and exit the file. Apply the CR. 
+```
+oc apply -f openstackcontrolplane.yaml
+```
+* Verify the pod is running.
+
+```
+oc get pods -n openstack | grep powerscale
+manila-share-powerscale-0                                         2/2     Running     0             21h
+```
+
+### Test the configured Backend
+
+* Ensure Manila share service is up and running.
+Open a session on the openstackclient pod:
+```
+oc rsh openstackclient
+```
+* Run the following command to check whether the manila services is up:
+```
+sh-5.1$ openstack share service list
++------------------+-----------------------+------+---------+-------+----------------------------+
+| Binary           | Host                  | Zone | Status  | State | Updated At                 |
++------------------+-----------------------+------+---------+-------+----------------------------+
+| manila-scheduler | hostgroup             | nova | enabled | up    | 2025-12-04T17:06:48.034510 |
+| manila-share     | hostgroup@powerscale  | nova | enabled | up    | 2025-12-04T17:06:46.526608 |
++------------------+-----------------------+------+---------+-------+----------------------------+
+```
+
+* Create a Share Type
+Finally, create a PowerScale share type and test if you can successfully create and attach shares of that type.
+Create a share type name powerscale:
+```
+openstack share type create --driver-handles-share-servers false powerscale
+```
+Map it to backend:
+```
+openstack share type set --property share_backend_name=powerscale powerscale
+```
+Confirm the share type is exits and is successfully configured.
+```
+sh-5.1$ openstack share type show powerscale
++----------------------+-------------------------------------------+
+| Field                | Value                                     |
++----------------------+-------------------------------------------+
+| id                   | 6c7c23ea-926c-4158-918b-5fcef1c73cc4      |
+| name                 | powerscale                                |
+| visibility           | public                                    |
+| required_extra_specs | driver_handles_share_servers : False      |
+| optional_extra_specs | share_backend_name : powerscale           |
+| description          | None                                      |
++----------------------+-------------------------------------------+
+```
+* Validate a Share Type
+Create a volume using the type created above without error to ensure the availability of the backend.
```

Comment:
Create a volume using the type created above and ensure there is no error to confirm the availability of the backend.